platform/ios/src/MGLMapView.mm

```diff
@@ -270,7 +270,7 @@ @implementation MGLMapView
     CADisplayLink *_displayLink;
     BOOL _needsDisplayRefresh;
 
-    NSUInteger _changeDelimiterSuppressionDepth;
+    NSInteger _changeDelimiterSuppressionDepth;
```

Comment:
This was changed to `NSInteger` since there's an assert later in the code that checks for >= 0.